openpype/hosts/flame/api/plugin.py

```diff
@@ -690,6 +690,19 @@ class ClipLoader(LoaderPlugin):
         )
     ]
 
+    _mapping = None
+
+    def get_native_colorspace(self, input_colorspace):
```

Comment:
I would recommend change this method to classmethod so the values are kept in plugin if multiple representations are loaded at a time.